packages/rules/src/consistentVersions.ts

```diff
@@ -79,3 +83,57 @@ const ensurePackageIsCorrectVersion = (
     });
   }
 };
+
+const ensurePackageMatchesSomeVersion = (
```

Comment:
I'm concerned that the `fixer` for multiple accepted values will be contentious, so to start off, I prefer adding this separate code path that doesn't attempt to fix when there are multiple `acceptedPackageDependencyValues`